src/PIL/ImageDraw.py

```diff
@@ -296,15 +296,14 @@ def rectangle(self, xy, fill=None, outline=None, width=1):
             self.draw.draw_rectangle(xy, ink, 0, width)
 
     def rounded_rectangle(
-        self, xy, radius=0, fill=None, outline=None, width=1, corners=None
+        self, xy, radius=0, fill=None, outline=None, width=1, **kwargs
```

Comment:
Or something like this for more transparency? Better for `help(rounded_rectangle)` and so on.

```suggestion
        self, xy, radius=0, fill=None, outline=None, width=1, *, corners=None
```

Or even this?

```suggestion
        self, xy, radius=0, fill=None, outline=None, width=1, *, corners=(True, True, True, True)
```

(And update/delete the below to match.)

Reply:
I think `corners=(True, True, True, True)` is best. But what would the ordering be? Top left, top right, bottom left, bottom right?

Reply:
I've updated the commit to use the first suggestion.

As for the ordering, yes, that is the ordering I've used. It mirrors the [ordering in CSS for `border-radius`](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius).